pr-queue/index.js

```diff
@@ -0,0 +1,289 @@
+/*
+ * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+/* Check if the trigger was a PR */
+function is_pull_request() {
+  return "GITHUB_BASE_REF" in process.env;
+}
+
+/* Get the target branch to check for PRs against */
+function target_branch() {
+  if (is_pull_request()) {
+    return process.env.GITHUB_BASE_REF;
+  } else {
+    return process.env.GITHUB_REF;
+  }
+}
+
+/* Get the name of the PR branch that triggered the PR */
+function pr_branch() {
+  if (is_pull_request()) {
+    return process.env.GITHUB_HEAD_REF;
+  }
+}
+
+/* Split the repository name into owner and repository */
+function repo_split() {
+  const repo = process.env.GITHUB_REPOSITORY;
+  const split_index = repo.indexOf('/');
+  return [repo.substr(0, split_index), repo.substr(split_index + 1)];
+}
+
+/* Get the owner of the repository */
+function repo_owner() {
+  return repo_split(0);
```

Comment:
This should work, but there is also `github.repository_owner` which should be passed as context to the action. Not sure if it's worth changing, just pointing out that it exists.